tests/validation/converters/test_pandapower_converter_output.py

```diff
@@ -65,6 +96,21 @@ def test_generate_output():  # TODO: REMOVE THIS FUNCTION
     json_converter.save(data=output_data, extra_info=extra_info)
 
 
+def test_generate_output_3ph():  # TODO: REMOVE THIS FUNCTION
```

Comment:
When or why should this function be removed?

Reply:
We shall remove this in the clearing of todos task